config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml

```diff
@@ -503,8 +503,8 @@ presubmits:
               value: "true"
             - name: TEST_WINDOWS
               value: "true"
-            - name: WINDOWS_FLAVOR
```

Comment:
just remove following config since it's running on 2019 actually
```
            - name: WINDOWS_FLAVOR
              value: "containerd-2022"
```

Reply:
Done